<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GlobalSettingsDialog</class>
 <widget class="QDialog" name="GlobalSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2"/>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>397</width>
        <height>900</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>900</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>339</width>
         <height>354</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetMinAndMaxSize</enum>
        </property>
        <item>
         <widget class="QMaemo5ValueButton" name="syncButton">
          <property name="text">
           <string>Address Book Synchronization</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QMaemo5ValueButton" name="syncFreqButton">
          <property name="text">
           <string>Synchronization Frequency</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QMaemo5ValueButton" name="portButton">
          <property name="text">
           <string>Port</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QMaemo5ValueButton" name="voiceCodecButton">
          <property name="text">
           <string>Voice Codec</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="startOnBootCheckBox">
          <property name="text">
           <string>Start Yappari when phone boots</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="nicknamesCheckBox">
          <property name="text">
           <string>Show nicknames in conversations</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="shownumbersCheckBox">
          <property name="text">
           <string>Show phone numbers in contacts dialog</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="popupCheckbox">
          <property name="text">
           <string>Pop up window when first message is received</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="blueChecksCheckbox">
          <property name="text">
           <string>Show and send read receipts for individual chats</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="enterIsSendCheckbox">
          <property name="text">
           <string>Use Enter to send the message</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="importCheckbox">
          <property name="text">
           <string>Import received media into gallery</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="autoCloseEmoji">
          <property name="text">
           <string>Auto close emoji dialog after selection</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Automatic download
media if it's less than:   </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="automaticDownloadLineEdit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="inputMethodHints">
             <set>Qt::ImhDigitsOnly</set>
            </property>
            <property name="text">
             <string>100</string>
            </property>
            <property name="maxLength">
             <number>5</number>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>KBytes</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>buttonBox</zorder>
  <zorder></zorder>
  <zorder>scrollArea</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QMaemo5ValueButton</class>
   <extends>QPushButton</extends>
   <header>qmaemo5valuebutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>GlobalSettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>GlobalSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
